ssz/utils.py

```diff
@@ -145,7 +145,7 @@ def _get_merkleized_result(
     max_depth: int,
     cache: CacheObj,
 ) -> Tuple[Hash32, CacheObj]:
-    merkleized_result_per_layers = [None for _ in range(max_depth + 1)]
+    merkleized_result_per_layers = [b"" for _ in range(max_depth + 1)]
```

Comment:
change is to satisfy the line `key = merkleized_result_per_layers[layer] + node` below. `merkleized_result_per_layers[layer]` is never actually `None` when that line is reached, but setting the list full of `b""` instead has the same effect and satisfies mypy.